source/installation-guide.rst

```diff
@@ -171,7 +169,7 @@ documentation`_.
 Kubernetes Requirements
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-- The Kubernetes supported version is **v1.11**.
+- The Kubernetes supported version is between **v1.11** and **v1.15** .
```

Comment:
Why do we support these specific versions only? It sounds a bit strange, in a previous paragraph, you have said another requirement.